### PR TITLE
Dark mode implemented and other UI tweaks

### PR DESCRIPTION
**Pull request by stigzler**
Hi. I've managed to get dark mode working and introduced some other UI tweaks:

1. Altered the log-in form to be smaller and to display a custom webpage, just confirming log-in. also had it pause a second rather than just flick up (unreadable). Pic attached. The prompt for a un/pw fits it well too. Feel free to use the page in the master branch as I won't be touching it: https://stigzler.github.io/
2. Updated the top icons and enlarged the buttons because they were a little small. Stuck with the VS icons. 
3. Detecting DarkMode was tricky. I couldn't find a way to do this through the VS ecosystem, so initially examined the registry to see if darkmode was set at a system level. Decided against this in the end and opted for a simple toggle button. When toggled, The user gets a msg at the bottom of the window saying changed will be applied on restart. 

Pics:
Screenshot 2024-04-04 065245
Screenshot 2024-04-04 065522

And now the bad news. I'm rubbish at WPF! So this is all really hacky, I'm afraid. Thus, do with it what you will. Hoping it might prove useful as a guide if you choose to update the xaml etc to be more WPF. 

Lastly, there were some really weird diff files - some in AppData/Temp. No idea how these were being included in the commit, and couldn't get rid of them either. Did as much as I could with the diff tidy. 

Oh, and thanks for making an awesome extension!

### CODE REVIEW

**Comment by conwid** [comment]
Thanks for the great feature! I'll take a look at the code in the PR as soon as I have some time and see what and how I can merge.

**Comment by stigzler** [comment]
Hey. Thanks for the response + the awesome extension! Sorry to mess you about, but I'm a hobby coder, so not a professional developer! It might be wise to hold off on this, as I'm working on some other features (see attached gif). I'm also going to revisit my code to see if I can make it 'more WPF' via resource dictionaries for themes etc. I'm just trying to figure how to manage branches to make it easier to merge if you wish. For the new features, I took a branch off the "DarkMode" branch. My history is looking like this in VS Git Branch manager:

image

My thoughts are to create a new branch called "My Branches" off my Master branch, merge "Gist Details" into that to provide a baseline (which should also merge in Dark Mode?) and then going forward, merge any separate feature branches back into "My Branches" 🤷However this might be totally wrong! Any thoughts appreciated. 

Basically, I'm looking for a way to keep developing whilst retaining the ability to create pull requests on separate features. I'd totally understand if my coding's not up to scratch and you can't merge, or if you need it fully MVVM/Wpf compliant, but please do let me know. I'm a bit of a grifter!

Oh and new "details view" feature (again, sorry - not done via MVVM):

GistEditorDetailsPreview

**Comment by conwid** [comment]
The idea should be the limit the pull requests to one feature. I'd probably keep the branches separate as much as possible and create a PO based on each branch, Merging a big PO might be a challenge in the future.
